src/zjs_dgram.c

```diff
@@ -134,7 +134,7 @@ static void udp_received(struct net_context *context,
     net_nbuf_unref(net_buf);
 
     jerry_value_t args[2] = {buf_js, rinfo};
-    zjs_signal_callback(handle->message_cb_id, args, 2);
+    zjs_signal_callback(handle->message_cb_id, args, sizeof(args));
```

Comment:
For arrays, like this it's much better to use sizeof than the magic number two. And in this case I'm fixing the bug I introduced recently with #842.